pom.xml

```diff
@@ -47,6 +55,8 @@
 
         <sonar.organization>green-code-initiative</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+        <sonar.pluginSourcesUrl>https://github.com/green-code-initiative/ecoCode/tree/main/${project.artifactId}</sonar.pluginSourcesUrl>
```

Comment:
this `sonar.pluginSourcesUrl` is incorrect because of 404 page (when we click on link on Administration tab, then installed tab) as mentionned above